### DISCUSSION TITLE
Filter the donation table in the New View

### DISCUSSION THREAD

**Discussion by Genevieve-K** [impress-org/givewp#7622] [Developer Q&A]
Hi. I have added custom fields for a donation and I would like to see them in the donation table in the admin dashboard. I added a filter in the Legacy View. How can I add the same filter in the new view?

**Comment by pauloiankoski** [comment]
Hi @Genevieve-K ! I wrote a snippet with an example of how to add a custom column when building the new list tables. Could you check if it works for you? [Link to snippet](https://github.com/impress-org/givewp-snippet-library/blob/master/admin-functions/add-table-list-column.php)

**Reply by Genevieve-K** [reply]
Thank you, it works perfectly.
